### DISCUSSION TITLE
Html Canvas or Svg Rendering

### DISCUSSION THREAD

**Discussion by Celibioux** [f5/unovis#361] [Q&A] [answered]
Hello Folks 👋
I Just found out about this amazing Charting goodie ! I Just can't find out what is use for rendering the charts please ? Like in Apache/Echarts you have the ability to chose between Canvas or Svg Rendering ! Thank you in advance

**Comment by rokotyan** [comment, marked as answer]
Thanks @Celibioux!

Can you tell us more about why you would like to see canvas rendering support? At the moment we don't have it in our plans because it would be a lot of work to add it. And in terms of performance, it will still be a compromise, as Canvas is much slower than, say, WebGL.

**Reply by Celibioux** [reply]
Thank you @rokotyan for your answer and i don't need html Canvas at all ! Just discovered your nice Charting tool thanks to Nuxt UI and i wish to use it in a project and i was afraid that it uses Canvas as it is slower than Svg Rendering ( in my use case) and i came to ask here because nothing on the docs explain your rendering mecanism and thank you again

**Reply by rokotyan** [reply]
@Celibioux Nice, glad to hear that, you're welcome!
